bookkeeper-server/src/test/java/org/apache/bookkeeper/conf/TestBKConfiguration.java

```diff
@@ -61,9 +61,9 @@ public static ServerConfiguration newServerConfiguration() {
         confReturn.setProperty(DbLedgerStorage.WRITE_CACHE_MAX_SIZE_MB, 4);
         confReturn.setProperty(DbLedgerStorage.READ_AHEAD_CACHE_MAX_SIZE_MB, 4);
         /**
-         * if testcase has zk error,just try 0 time for fast running
+         * if testcase has zk error,just try 3 time for fast running
          */
-        confReturn.setZkRetryBackoffMaxRetries(0);
+        confReturn.setZkRetryBackoffMaxRetries(3);
```

Comment:
Before this PR https://github.com/apache/bookkeeper/pull/2958, the ZK default retry count is `Integer.MAX_VALUE`,   I wonder why not use `Integer.MAX_VALUE` instead of 3?

Reply:
@hangc0276  set zkRetryBackoffMaxRetries to Integer.MAX_VALUE is ok, I have changed it ,thanks
<img width="2496" alt="image" src="https://user-images.githubusercontent.com/42990025/176818886-0a47bd6b-ceb0-4c5c-b60c-fcd430930c7e.png">

Reply:
Removing this configuration is better because the default value is Integer.MAX_VALUE

Reply:
ok,add a new commit @hangc0276 
    1.The serial execution to close more auditor is easy timeout,change it concurrent processiing
    2.Removing config's zkRetryBackoffMaxRetries,use the default value is Integer.MAX_VALUE